docs/modules/ROOT/attachments/HardwareDetailsRunFacet.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
S: I'm wondering if these should live in the codebase, then get copied here?  

Imagine a scenario where you don't have access to the internet - might need to be able to get at them locally or extract them to another, available hosting location as well?

If so, can definitely be another ticket.

Reply:
We determined in the previous ticket that these schemas only serve as supplemental documentation of your custom facets. They are not technically needed for your lineage to function correctly in downstream projects, so your local project does not need direct access to these schemas.

Reply:
For clarification, the big realization was that the schemas of OL are at the very least under-utilized.  From a cursory search the major tools that support OL don't interact with schemas at all (hence calling them documentation).  This may be due to the fact that OL doesn't really push you to make schemas for custom facets and implies that the most common approach is to omit schema info.  So until major tooling starts to support and leverage schema definitions of custom facets from libraries like ours, there isn't a ton of demand for edge cases like the one you're describing.

Reply:
As I tried to note below (sorry - email response got misaligned), it just feels odd for us to provide a schema that we don't even use.  Feels like we should a least run the validation ourselves if we are going to provide it.

FWIW, I think most users would get more mileage out of the straight documentation of an example json file than reading a schema.   So this might be where the rub is for me.

Reply:
Agreed that the current schema's aren't particularly useful in their current state. We do have documentation outlining all our custom facets and their inputs [here](https://boozallen.github.io/aissemble/aissemble/current/model-lineage.html#_facet). Could be worthwhile to actually use our schemas for actually validating our python implementation in another ticket.